initializr-generator/src/main/java/io/spring/initializr/generator/ProjectGenerator.java

```diff
@@ -95,6 +95,48 @@
 
 	private transient Map<String, List<File>> temporaryFiles = new LinkedHashMap<>();
 
+	private final static Map<String, List<String>> annotationMap = new HashMap<>();
+
+	private final static Map<String, List<String>> importMap = new HashMap<>();
+
+	static {
+		final String azureServiceBus = "azure-service-bus";
+		final String cloudConfigServer = "cloud-config-server";
+		final String cloudEurekaServer = "cloud-eureka-server";
+		final String cloudGateway = "cloud-gateway";
+		final String cloudHystrixDashboard = "cloud-hystrix-dashboard";
+
+		// Azure Service Bus
+		annotationMap.put(azureServiceBus, Arrays.asList("@SpringBootApplication", "@EnableDiscoveryClient"));
+		importMap.put(azureServiceBus, Arrays.asList(
+				"org.springframework.boot.autoconfigure.SpringBootApplication",
+				"org.springframework.cloud.client.discovery.EnableDiscoveryClient"));
+
+		// Cloud Config Server
+		annotationMap.put(cloudConfigServer, Arrays.asList("@SpringBootApplication", "@EnableDiscoveryClient"));
```

Comment:
does config server require @EnableConfigServer?

Reply:
yeah, should be `@EnableConfigServer` instead of `@EnableDiscoveryClient`. Forgot to replace that.